dev-tools/src/main/resources/plugin-metadata/plugin-descriptor.properties

```diff
@@ -64,6 +64,9 @@ classname=${elasticsearch.plugin.classname}
 java.version=${maven.compiler.target}
 #
 # 'elasticsearch.version' version of elasticsearch compiled against
+# You will have to release a new version of the plugin for each new
+# elasticsearch release. This version is checked when the plugin
+# is loaded so non-compatible plugins are not loaded.
```

Comment:
Actually, Elasticsearch will refuse to start in the presence of plugins with the incorrect `elasticsearch.version`.